.ci/targets/windows_build_and_platform_tests.yaml

```diff
@@ -2,6 +2,10 @@ tasks:
   - name: prepare tool
     script: .ci/scripts/prepare_tool.sh
     infra_step: true # Note infra steps failing prevents "always" from running.
+  - name: download Dart deps
+    script: script/tool_runner.sh
```

Comment:
This name is kind of awkward but I genuinely can't think of something better. I thought filter might be better but this is the opposite of a filter is is an allow list.

Reply:
The tool unfortunately collects these useful-but-hard-to-describe-succinctly flags. This was my third or fourth attempt, and it was the best I came up with as well.